### PR TITLE
KelvinVR: Working library with LED methods

### PR DESCRIPTION
**Pull request by NaturalAutomatics**
Created working library with ESP32/Arduino platform and LED methods for testing

### CODE REVIEW

File: MetaTempDriver/Hardware/library/KelvinVR/README.md
Comment (NaturalAutomatics) [inline MetaTempDriver/Hardware/library/KelvinVR/README.md:1]: Change REAMDE! This one is old

File: MetaTempDriver/Hardware/library/KelvinVR/library.json
Comment (NaturalAutomatics) [inline MetaTempDriver/Hardware/library/KelvinVR/library.json:1]: [nit] Upload library here

File: MetaTempDriver/Hardware/library/KelvinVR/library.properties
Comment (NaturalAutomatics) [inline MetaTempDriver/Hardware/library/KelvinVR/library.properties:1]: Fill all the data required for lib here

**Review by NaturalAutomatics — COMMENTED** [review]
All library code was reviewed and comments added

File: MetaTempDriver/Hardware/library/KelvinVR/README.md
Comment (AlexNoyanov) [reply MetaTempDriver/Hardware/library/KelvinVR/README.md:1]: README updated

File: MetaTempDriver/Hardware/library/KelvinVR/library.properties
Comment (AlexNoyanov) [reply MetaTempDriver/Hardware/library/KelvinVR/library.properties:1]: Done!

**Pull request merged**